pyproject.toml

```diff
@@ -120,8 +140,7 @@ extend-select = [
     "S101",  # allow assertions in tests
     "T201",  # print allowed
 ]
-"symusic/core.pyi" = [
-    "UP007", # `X | Y` for type annotations
+"python/symusic/core.pyi" = [
```

Comment:
With future annotation imports it works for python < v3.10, so it's now working with | and future proof